Tasks/WindowsMachineFileCopy/RoboCopyJob.ps1

```diff
@@ -161,16 +161,31 @@ param (
         return ""
     }
     
+    function Get-NetExeCommand
+    {
+        $netExePath = Join-Path -path (get-item env:\windir).value -ChildPath system32\net.exe
+        if(Test-Path $netExePath)
+        {
+            Write-Verbose "Found the net exe path $netExePath. Net command will be $netExePath"
+            return $netExePath
+        }
+        
```

Comment:
no need, but I will have a bug to add tests for "RoboCopyJob.ps1".  I am not able to find any existing tests covering the logic with "RoboCopyJob.ps1"